sources/packages/backend/apps/db-migrations/src/sql/DisbursementFeedbackErrors/Add-column-ecert-feedback-error-id.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE
+    sims.disbursement_feedback_errors
+ADD
+    COLUMN ecert_feedback_error_id INT REFERENCES sims.ecert_feedback_errors(id);
+
+COMMENT ON COLUMN sims.disbursement_feedback_errors.ecert_feedback_error_id IS 'E-Cert feedback error id of the error received.';
+
+-- Populate ecert_feedback_error_id based on existing column error_code.
+UPDATE
+    sims.disbursement_feedback_errors
+SET
+    ecert_feedback_error_id = errors.id
+FROM
+    sims.ecert_feedback_errors errors
+WHERE
+    errors.error_code = sims.disbursement_feedback_errors.error_code
+    AND errors.offering_intensity = (
+        SELECT
+            epo.offering_intensity
+        FROM
+            sims.disbursement_schedules ds
```

Comment:
Minor but most of the time we use the aliases declared in a more intended way, for instance, in SQL files under `libs\src-sql`.